### ISSUE TITLE
Hdfs behavior changes

### ISSUE THREAD

**Issue by f4hy** [fsspec/filesystem_spec#1086]
The recent release of fsspec has broken some of our workflows. We were accessing hdfs through fsspec and operations like 
```
fs = fsspec.filesystem("hdfs")
fs.ls('hdfs:///path/on/hdfs/")
```
work on versions `fsspec<2022.10.0` but when using the `2022.10.0` release this operation errors with 

`ArrowInvalid: FileSelector.base_dir must not be a URI, got: hdfs:///path/on/hdfs/`

Most likely due to changes in #1051 . I will try to investigate why exactly this broke.

**Comment by martindurant** [comment]
Would you mind trying a couple of similar URLs, like
```
fs.ls('hdfs://path/on/hdfs/")
fs.ls('/path/on/hdfs/")
```
?

Unfortunately, the message from arrow is not very helpful here.

**Comment by f4hy** [comment]
The second one there works for me. so Yes if we specify an hdfs filesystem and then use non uri paths, it works. 

I wonder why the uri paths broke. We could rework our workflows to strip the `hdfs://` before calling , but that feels like the wrong solution.

**Comment by leoleoasd** [comment]
Fixed in #1099 
@martindurant should we close this issue?
